src/app/component/Header/Header.container.js

```diff
@@ -229,20 +251,21 @@ export class HeaderContainer extends NavigationAbstractContainer {
     }
 
     onSearchOutsideClick() {
-        const {
-            goToPreviousNavigationState,
-            hideActiveOverlay,
-            navigationState: { name }
-        } = this.props;
+        const { goToPreviousNavigationState, navigationState: { name } } = this.props;
 
         if (!isMobile.any() && name === SEARCH) {
-            this.setState({ searchCriteria: '' });
-
-            hideActiveOverlay();
+            this.hideSearchOverlay();
             goToPreviousNavigationState();
         }
     }
 
+    goToDefaultHeaderState() {
```

Comment:
Maybe worth moving into abstract navigation? Sounds pretty logical to fallback onto default route for the pathname.

Reply:
Moved to abstract navigation.